l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py

```diff
@@ -378,6 +384,9 @@ def _onchange_fiscal_operation_id(self):
 
     @api.onchange("fiscal_operation_line_id")
     def _onchange_fiscal_operation_line_id(self):
+        if (self.fiscal_operation_line_id.tax_calc == TAX_CALC_MANUAL or
```

Comment:
Tem que testa se a linha da operação fiscal existe senão vai ter um erro https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py#L384, isso é importante quando for criado uma fatura não fiscal por exemplo...